More specific and slightly gentler information about the preview period (#415)
apollographql/apollo-cli: docs/source/index.mdx

import { Link } from 'gatsby';
import { colors } from 'gatsby-theme-apollo-core';

> > **Rover is in active, rapid development.** Rover commands are subject to breaking changes without notice. Until we declare version 0.1.0 or higher, we do not yet recommend integrating Rover in critical production systems.

Reviewer: "we declare" -> "the release of"

Author: Done

> > **Rover is in active, rapid development.** Rover commands are subject to breaking changes without notice. Until we declare version 0.1.0 or higher, we do not yet recommend integrating Rover in critical production systems.

Author: ```suggestion
> **Rover is in active, rapid development.** Rover commands are subject to breaking changes without notice. Until the release of version 0.1.0 or higher, we do not yet recommend integrating Rover in critical production systems.
```

## Public preview

> Rover's public preview period is currently active! Throughout this period, the Rover team will release new preview versions of Rover for the public to test out and provide feedback on. We plan to release a new version approximately every 2 weeks, with the possibility of critical bug patches in between. While we no longer have any planned changes to the command structure, rior to version 0.1.0 we reserve the right to make breaking changes in new releases without notice.

Author: ```suggestion
Rover's public preview period is currently active! Throughout this period, the Rover team will release new preview versions of Rover for the public to test out and provide feedback on. We plan to release a new version approximately every 2 weeks, with the possibility of critical bug patches in between. While we no longer have any planned changes to the command structure, prior to version 0.1.0 we reserve the right to make breaking changes in new releases without notice.
```

> Rover's public preview period is currently active! Throughout this period, the Rover team will release new preview versions of Rover for the public to test out and provide feedback on. We plan to release a new version approximately every 2 weeks, with the possibility of critical bug patches in between. While we no longer have any planned changes to the command structure, prior to version 0.1.0 we reserve the right to make breaking changes in new releases without notice.

Reviewer: "While" -> "Although"

Author: Changed

> Rover's public preview period is currently active! Throughout this period, the Rover team will release new preview versions of Rover for the public to test out and provide feedback on. We plan to release a new version approximately every 2 weeks, with the possibility of critical bug patches in between. While we no longer have any planned changes to the command structure, prior to version 0.1.0 we reserve the right to make breaking changes in new releases without notice.

Author: ```suggestion
Rover's public preview period is currently active! Throughout this period, the Rover team will release new preview versions of Rover for the public to test out and provide feedback on. We plan to release a new version approximately every 2 weeks, with the possibility of critical bug patches in between. Although we no longer have any planned changes to the command structure, prior to version 0.1.0 we reserve the right to make breaking changes in new releases without notice.
```
